packages/vite/src/node/env.ts

```diff
@@ -35,35 +35,38 @@ export function loadEnv(
     }
   }
```

Comment:
Sorry, I just noticed `VITE_*` env variables cannot be used in `.env`.
```env
# .env
VITE_FOO=$VITE_BAR
```
```shell
VITE_BAR=bar vite dev
```

Reply:
vue-cli supported it?

Reply:
I don't know about it. But I noticed that this wasn't working and should be done while fixing #6626.